extension/js/agent/patches/patchBackboneExtend.js

```diff
@@ -0,0 +1,16 @@
+;(function(Agent) {
+  
+  Agent.patchBackboneExtend = function(BackboneClass) {
+  
+    Agent.setHiddenProperty(BackboneClass.prototype, 'classId', _.uniqueId('class'));
```

Comment:
You mean "if not already set?"  Like in case `patchBackboneExtend` is called multiple times on the same prototype?